index.rst

```diff
@@ -10,20 +10,849 @@
+Each of the use cases follow a standardized `template <https://confluence.lsstcorp.org/display/LSSTCOM/On-the-fly+Analysis+Use-Case+Template>`_ which differs slightly from that which was used in the first FAFF charge.
+
+The remaining use-cases for FAFF2 can be found on the FAFF use-cases page `on confluence <https://confluence.lsstcorp.org/display/LSSTCOM/Use-Cases>`_ and are referenced throughout the remainder of this report.
+
+Daytime Calibration
+^^^^^^^^^^^^^^^^^^^
+
+.. warning::
+
+   This section is not yet completed and only provides a status update.
+
+
+During the course of the working group, the example of daytime calibration was raised repeatedly, specifically in regards to how calibration data products are generated and what is expected of the observing specialist.
+The aspect pertaining specifically to the FAFF charge is what the observer is required to look at during the process, including both images and/or alarms.
+The details of how Daytime Calibration is performed is being documented in `DMTN-222 <https://DMTN-222.lsst.io>`_ and will not be repeated as a new FAFF use-case.
+
+In short, a SAL script is launched by the observer to acquire a daytime set of calibrations.
+This SAL script launches an OCPS-based processing of the images, but the ScriptQueue does not block on the processing awaiting the final analysis.
+Currently, if the process fails then no alert is generated automatically.
+However, as will be discussed in the following sections, a Watcher [#]_ alarm will be setup to listen and alert users (via LOVE) in the event of a catastrophic failure in the analysis which the observer could do something about (e.g. the shutter did not open and the flats have no signal).
+How the observer responds to the alert is currently being discussed.
+Presumably, this will use a parameterized notebook that will allow an observer to better understand the issue.
+Any viewing of the raw frames themselves will utilize the Camera Visualization Tool.
+
+In the case where a more complex issue arises (e.g., a 2% increase in bad pixels is observed), this is addressed by the calibration team offsite and is not immediately reported to the summit team.
+When the calibrations used on the summit need to be updated, this is the role of the calibration scientist and is not the responsibility of the observer.
+Furthermore, this cadence is expected to be slow (months) and is therefore outside the scope of this charge.
+
+.. [#] The `Watcher CSC <https://ts-watcher.lsst.io/>`_ is provided a list of "rules" that it ensures the system is always obeying. If a rule is violated, such as a temperature going out of specification, the an alert or alarm is issued to the observer via the LOVE interface. The alarm stays in place until the rule is no longer violated and the original alert has been acknowledged. The Watcher is not able to perform analyses and only evaluates simple conditions.
```

Comment:
the an alert -> an alert

This footnote is not following the "semantic line break" rule. All phrases are in a single line